### PR TITLE
Sortable and language specific search documents

### PR DESCRIPTION
**Pull request by ammarghaus**

### CODE REVIEW

File: lib/pg_search/multisearchable.rb [hunk 1]
```diff
@@ -6,60 +6,81 @@ module PgSearch
...
-        after_save :update_pg_search_document,
+        after_save :update_pg_search_documents,
                    if: -> { PgSearch.multisearch_enabled? }
+
+        after_destroy :clear_search_documents
       end
     end
 
-    def searchable_text
+    def searchable_text(language)
```
Comment (bvangennep) [inline lib/pg_search/multisearchable.rb]: Maybe we should do this with a default value like we do in other places?

``
def searchable_text(language: I18n.locale)
end

File: lib/pg_search/multisearchable.rb [hunk 2]
```diff
@@ -6,60 +6,81 @@ module PgSearch
...
     end
 
-    def pg_search_document_attrs
-      {
-        content: searchable_text
-      }.tap do |h|
-        if (attrs = pg_search_multisearchable_options[:additional_attributes])
-          h.merge! attrs.to_proc.call(self)
+    def search_languages
+      pg_search_multisearchable_options[:languages]&.to_proc&.call(self) || [I18n.default_locale]
```
Comment (bvangennep) [inline lib/pg_search/multisearchable.rb]:
```suggestion
      pg_search_multisearchable_options[:languages]&.to_proc&.call(self) || I18n.available_locales
```

**Review by bvangennep — CHANGES_REQUESTED** [review]

File: lib/pg_search/multisearchable.rb (hunk 2 above)
Comment (ammarghaus) [reply lib/pg_search/multisearchable.rb]: Fixed in language:

Comment (ammarghaus) [reply lib/pg_search/multisearchable.rb]: Fixed in 29f753c0d513d39ec44745724ca48d8a63833230

File: lib/pg_search/multisearchable.rb (hunk 1 above)
Comment (ammarghaus) [reply lib/pg_search/multisearchable.rb]: Fixed in 29f753c0d513d39ec44745724ca48d8a63833230

**Review by bvangennep — APPROVED** [review]

**Pull request merged**